src/polytopes/ngon.jl

```diff
@@ -160,11 +160,22 @@ function normal(t::Triangle{3})
   n / norm(n)
 end
 
+function (t::Triangle)(u::T, v::T) where T
+    w = (one(T) - u - v)
+    if u < zero(T) || u > one(T) ||
+       v < zero(T) || v > one(T) ||
+       w < zero(T) || w > one(T)
+       throw(DomainError("barycentric coordinates out of range"))
+    end
+    v₁, v₂, v₃ = coordinates.(t.vertices)
+    Point(v₁*u + v₂*v + v₃*w)
```

Comment:
Now that I am seeing the behavior in the tests, I think it is more interesting to use the last two vertices as reference so that the coordinate `(u,v) = (0,0)` is in the "bottom left", the coordinate `(u,v) = (1,0)` is in the "bottom right" and `(u,v) = (0,1)` is in the "top". Can you please adjust this? It will be more intuitive compared to the quadrangle case.

Reply:
You are right. I will change it.